router/jwt.go

```diff
@@ -0,0 +1,29 @@
+package router
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go/request"
+	"github.com/gin-gonic/gin"
+)
+
+// Auth authenticates request validating JWT signature by a given public key.
+// a user agent must provide a BEARER token via the Authorization header
+func Auth() gin.HandlerFunc {
+	publicKeyPEM, _ := ioutil.ReadFile(os.Getenv("JWT_PUBLIC_KEY_PATH"))
```

Comment:
Yes, that's what I'm going to do next since I will have the same problem when deploying it.